pkg/ext-proc/backend/inferencemodel_reconciler.go

```diff
@@ -25,7 +25,7 @@ func (c *InferenceModelReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	if req.Namespace != c.PoolNamespacedName.Namespace {
 		return ctrl.Result{}, nil
 	}
-	klog.V(1).Info("reconciling InferenceModel", req.NamespacedName)
+	klog.V(1).Infof("reconciling InferenceModel %v", req.NamespacedName)
```

Comment:
can you use the logging levels defined at https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/main/pkg/ext-proc/util/logging/logging_const.go

It would be great if you can update that for all logging in this file :)

Reply:
Hi, I actually noticed only after this issue that there is #251 . I would honestly just merge this quickly and move on to work on complete migration to the logging guidelines, if you don't mind.